### DISCUSSION TITLE
Repository with 99 stars didn't receive a badge

### DISCUSSION THREAD

**Discussion by Devanshu-17** [community/community#53219] [Other Feature Feedback, Questions, & Ideas] [answered]
## Hello. I have a repository on which I have around 99 stars, but I still haven't received the Starstruck Badge.

This is the repository in question:

https://github.com/Devanshu-17/Mac_Spotify_Adblock

**Comment by hamdivazim-archive** [comment]
Hi, this seems to be a bug with GitHub. I think you should [contact GitHub support](https://support.github.com) and ask them about the issue.

**Reply by Devanshu-17** [reply]
Hey, thanks for replying. I have raised a ticket and will hopefully get an answer back soon.

**Comment by NJacksonDev** [comment]
Hmm, that is odd. Maybe Github support can answer. 

The below unofficial repos also have some information about how to qualify for the various badges (with 128 stars you should get the bronze Starstruck).

https://github.com/Schweinepriester/github-profile-achievements

https://github.com/drknzz/GitHub-Achievements

**Comment by YasinDehfuli** [comment]
you need 128 stars to get X2 Level of Starstruck badge

**Comment by Devanshu-17** [comment, marked as answer]
Hey, everyone. Thanks for replying. I raised an issue at Github Support and my query was resolved. Now, I have received the Starstruck badge on my profile
